src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpTraceHelper.cs

```diff
@@ -33,7 +33,27 @@ public static bool IsTraceEnabled()
             return s_TraceEnabled;
         }
 
-        public static void TraceCallbackStatus(string message, IntPtr handle, bool invokeCallback, uint status)
+        public static void Trace(string message)
+        {
+            if (!IsTraceEnabled())
+            {
+                return;
+            }
+            
+            Debug.WriteLine(message);
+        }
+
+        public static void Trace(string format, object arg0, object arg1, object arg2)
```

Comment:
Good points. I'll wrap the call site for now. Although the wrapping code doesn't look as good, it will serve as a reminder to keep perf good when I'm converting this to the final tracing logic